app/components/settings/project_custom_fields/edit_form_header_component.rb

```diff
@@ -34,6 +34,11 @@ def initialize(custom_field:)
 
         @custom_field = custom_field
       end
+
+      def breadcrumbs_items
+        [{ href: admin_settings_project_custom_fields_path, text: t("label_administration") },
```

Comment:
I think this definition is not correct. Since this is an Edit page, I expect a three parts breadcrumb:

<img width="233" alt="Bildschirmfoto 2024-04-02 um 09 04 41" src="https://github.com/opf/openproject/assets/7457313/fb02fa9e-9f97-4580-aaab-2707c10573e7">